www/html/ci/application/controllers/Landing.php

```diff
@@ -191,6 +201,16 @@ public function uploadResume(){
 			echo "\nFile not uploaded";
 		} elseif (move_uploaded_file($filetmp, $targetFile)) {
 			echo "File Uploaded";
+
+			$this->db->where('filename', $oldfile);
+			$this->db->delete('resume_file');
```

Comment:
Are you deleting the whole table and then creating it again? If so, you should just be inserting rows instead.